lib/ebpd_loader.cc

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <string>
+#include "lib/ebpd_utils.h"
+#include "lib/ebpd_loader.h"
+#include "lib/ebpd.h"
+
+using namespace std;
+
+int
+EbpdLoader::LoadFile(const char *filepath, int ifindex) {
+    int ret;
+    ret = ebpd_load_xdp_prog(filepath, ifindex, &handle);
+    if (ret) {
+        cout << "Error: eBPF program " << filepath << " load failed " << ret << "\n";
+        return ret;
+    }
+    cout << "eBPF program load succeeded, obj: " << handle << "\n";
+    return 0;
+}
+
+int
+EbpdLoader::LoadBuffer(void *buf, int buf_size, const char *name) {
```

Comment:
"const std::string_view buf" later on buf.data() and buf.size() can be used in place of buf and buf_size.
"const std::string& name" used name.c_str() later

Reply:
Done